css/port.css

```diff
@@ -0,0 +1,297 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+.bk_container {
+  background-color: #e5e5e5;
+  max-width: 375px;
+  margin: auto;
+}
+
+.port_container {
+  max-width: 375px;
+  height: 100vh;
```

Comment:
- [ ] If at all you have to use fixed height, you should use `min-height: 100vh` instead. This way, it make the section to be at least 100vh if there is no content, but as more content are added it allows the section to grow downwards.